### PR TITLE
Added linters

### PR DESCRIPTION
**Pull request by BiassedTomato**
## Summary

Added Prettier, ESLint, ESLint-Prettier config, Husky pre-commit hook, lint-staged.

Now if there are style errors in the code, you are forced to fix them before you can proceed with committing.

### Screenshot
image

## How this was tested?

Manually

### List of steps to manually test introduced functionality:
- Add `'debugger;'` to `src/App.jsx` below imports;
- ESLint VSCODE extension should highlight it as "unexpected 'debugger' statement."
- `git add .`
- `git commit -m "tested pre-commit hook"`
- This should cause an error; the console will output the error location (see screenshot.)
- As such, the files will not be committed until the developer solves all the errors that ESLint finds.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have added tests for new functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] My pull-request is shorter than 500 lines

### CODE REVIEW

**Comment by BiassedTomato** [comment]
There is currently a conflict in src/App.js;

When this branch was created, App.js had boilerplate code in it. In one of the more recent PRs Evgenia removed it.
In this I changed the extension to .jsx as per linter rules and left the boilerplate from CRA. My idea to fix this is to merge it with the version in `main` and create another PR to change it to .jsx;

Alternatively, I can remove boilerplate code myself and choose my version over Evgenia's.

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
```
Comment (nadezhdakharchuk) [inline .eslintrc.js:1]: Could you please remove this duplicated file

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (nadezhdakharchuk) [inline .eslintrc.json:1]: Could you please rename this file to `.eslintrc`

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "env": {
```
Comment (nadezhdakharchuk) [inline .eslintrc.json:2]: Could you please use 2 space indentation for this file

File: .prettierrc.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (nadezhdakharchuk) [inline .prettierrc.json:1]: Could you please add come rules for prettier
For example:
```
{
  "singleQuote": true,
  "printWidth": 120,
  "semi": true,
  "trailingComma": "all"
}
```

File: package.json [hunk 5]
```diff
@@ -17,13 +17,8 @@
...
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+    "eject": "react-scripts eject",
+    "prepare": "husky install"
```
Comment (nadezhdakharchuk) [inline package.json:21]: Could you please add script for linting my code

**Pull request closed without merging**